test.js

```diff
@@ -25,23 +28,46 @@ test('falsey cases', t => {
 		'NO',
 		'No',
 		'false',
+		'FALSE',
+		'False',
 		false,
+		'0',
 		0
 	].forEach(el => {
 		t.false(yn(el));
 		t.false(yn(el, {lenient: true}));
 	});
 });
 
-test('edge cases which return null', t => {
-	t.is(yn(NaN), null);
-	t.is(yn(''), null);
-	t.is(yn('yn'), null);
-	t.is(yn('unicorn'), null);
-	t.is(yn(NaN, {lenient: true}), null);
-	t.is(yn('', {lenient: true}), null);
-	t.is(yn('yn', {lenient: true}), null);
-	t.is(yn('unicorn', {lenient: true}), null);
+test('null cases', t => {
+	[
```

Comment:
It works, but since ES2015, we now of [`for...of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) to iterate over arrays which we hadn't before. So it's cleaner and best practice to use the `for...of` instead of that array method with a callback.

It should be `t.is(yn(el), null);` instead of `t.is(null, yn(el));` btw. It's first `actual` and then `expected`. See [docs](https://github.com/avajs/ava#isvalue-expected-message).